tutorials/03_finetune_a_reader.ipynb

```diff
@@ -0,0 +1,258 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fine-Tune a Reader\n",
+    "\n",
+    "- **Level**: Intermediate\n",
+    "- **Time to complete**: 20 minutes\n",
+    "- **Nodes Used**: `FARMReader`\n",
+    "- **Goal**: Learn how to improve the performance of a Reader model by performing fine-tuning."
```

Comment:
I'd make it more concrete -> I'd say what Reader model you're fine-tuning. Can we also say how much the performance improves?

Reply:
I have made it more concrete. However, it is not possible in this set up to say how much it improves because we are using the evaluation set to further train the model.